qa/suites/crimson-rados/rbd/tasks/rbd_api_tests.yaml

```diff
@@ -0,0 +1,21 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - overall HEALTH_
+      - \(CACHE_POOL_NO_HIT_SET\)
+      - \(POOL_APP_NOT_ENABLED\)
+      - is full \(reached quota
+      - \(OSDMAP_FLAGS\)
+      - \(OSD_
+      - \(PG_
+      - \(POOL_
+      - \(CACHE_POOL_
+      - \(POOL_FULL\)
+tasks:
+- workunit:
+    clients:
+      client.0:
+        - rbd/crimson/test_crimson_librbd.sh
+    env:
+      RBD_FEATURES: "1"
```

Comment:
Just curious, what is making you stick to `1` (just `layering` feature) here?  Have you tried enabling additional features?

FWIW the most widely used combination in the classic suite is `61`.

Reply:
Based on `test_librbd.cc` `get_features` function.
if `RBD_FEATURES` has any value set - we use new format, else we use old one
Therefore if we set `RBD_FEATURES ="1"` the new format will be tested.

This is also used in classic tests.

```
static int get_features(bool *old_format, uint64_t *features)
{
  const char *c = getenv("RBD_FEATURES");
  if (c && strlen(c) > 0) {
    stringstream ss;
    ss << c;
    ss >> *features;
    if (ss.fail())
      return -EINVAL;
    *old_format = false;
    cout << "using new format!" << std::endl;
  } else {
    *old_format = true;
    *features = 0;
    cout << "using old format" << std::endl;
  }

  return 0;
}
```

Reply:
Is the intention to test _just_ the new format or the feature set that comes with the new format?  To enable tests for what users get exposed to when they create an RBD image without specifying any additional options, `RBD_FEATURES` must be set to `61`.